src/formatDate.js

```diff
@@ -48,9 +48,25 @@
  *
  * @returns {string}
  */
-
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateSplit = date.split(fromFormat[3]);
+
+  return toFormat.slice(0, 3).reduce((accumulator, current, i) => {
```

Comment:
let's destructure:
```suggestion
  const [fromValue1, fromValue2, fromDelimiter] = fromFormat;
  const [toValue1, toValue2, toDelimiter] = toFormat;
  const dateSplit = date.split(fromDelimiter);

  return toFormat.slice(0, 3).reduce((accumulator, current, i) => {
```

Reply:
But I'd need the indexes in the ```fromFormat```.. I'll spend some time on it.

Reply:
I looked for a spread operation like 
```js
const [...fromValues, fromDelimiter] = fromFormat;
```
Apparently can't do that, let me push the solution according to the rest of the comments. Haven't been able to figure out a cool solution here, yet